spec/outputs/logentries_spec.rb

```diff
@@ -0,0 +1,2 @@
+require "logstash/devutils/rspec/spec_helper"
```

Comment:
Hi @purbon, it would be grat if you could put me into a right direction for testing. I was trying to carry out a "bundle install" in order to publish the gem to rubygems.org, however it's giving me:

<command line output>
Could not find gem 'logstash-devutils (>= 0) ruby' in the gems available on this machine.
<end>